tests/test_cmdline.py

```diff
@@ -520,11 +541,18 @@ def test_list_sources_detailed__from_both_python_file_and_stdin(fake_project):
         "--list-sources", f"{tmp_path}", "--code", f"{tmp_path}", "-", "--detailed"
     )
     expect = [
-        "Sources of Python code:",
-        f"  {tmp_path}/code.py (using {tmp_path}/ as base for 1st-party imports)",
-        "  <stdin>",
+        [
+            "Sources of Python code:",
+            f"  {tmp_path / 'code.py'} (using {tmp_path} as base for 1st-party imports)",
+            "  <stdin>",
+        ],
+        [
+            "Sources of Python code:",
+            "  <stdin>",
+            f"  {tmp_path / 'code.py'} (using {tmp_path} as base for 1st-party imports)",
+        ],
     ]
-    assert output.splitlines() == expect
+    assert output.splitlines() == expect[0] or output.splitlines() == expect[1]
```

Comment:
```suggestion
    assert output.splitlines() in expect
```

Reply:
Updated in 127473a053bfcf991c9a92f78a3e440f6d84cd0e